pkg/commands/compute/build.go

```diff
@@ -167,17 +167,7 @@ func (c *BuildCommand) Exec(in io.Reader, out io.Writer) (err error) {
 		return err
 	}
 
-	postBuildCallback := func() error {
```

Comment:
To fix the order of rendering I needed to move this logic (and associated prompt functionality) inside the toolchain implementation.